src/ontology/mondo-edit.obo

```diff
@@ -281805,15 +281803,18 @@ property_value: confidence "2.692307692307692" xsd:double
 id: MONDO:0014343
 name: Desbuquois dysplasia 2
 def: "Any Desbuquois dysplasia in which the cause of the disease is a mutation in the XYLT1 gene." [MONDO:patterns/disease_series_by_gene]
+synonym: "Baratela-Scott syndrome" EXACT [MONDO:0010470, OMIM:300881]
 synonym: "DBQD2" RELATED ABBREVIATION [MONDO:Lexical, OMIM:615777]
 synonym: "Desbuquois dysplasia 2" EXACT [MONDO:Lexical, OMIM:615777]
 synonym: "Desbuquois dysplasia caused by mutation in XYLT1" EXACT [MONDO:design_pattern]
 synonym: "Desbuquois dysplasia type 2" EXACT [MONDORULE:1, OMIM:615777]
 synonym: "XYLT1 Desbuquois dysplasia" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+xref: OMIM:300881 {source="MONDO:obsoleteEquivalent"}
 xref: OMIM:615777 {source="MONDO:equivalentTo"}
 xref: Orphanet:1425 {source="MONDO:subClassOf", source="OMIM:615777"}
+xref: UMLS:C3550876 {source="OMIM:300881", source="NCBI:mim2gene_medline", source="MONDO:equivalentTo"}
```

Comment:
can you make this MONDO:obsoleteEquivalent
we can't have two MONDO:equivalentTo, this is called a proxy merge